base/shell/explorer/notifyiconscust.cpp

```diff
@@ -20,31 +21,244 @@
 
 #include "precomp.h"
 
+VOID ShowBehaviorCombo(HWND hwnd, LPNMITEMACTIVATE itemdata)
+{
+    HWND hListView = GetDlgItem(hwnd, IDC_NOTIFICATION_LIST);
+    HWND hCombo = GetDlgItem(hwnd, IDC_NOTIFICATION_BEHAVIOUR);
+
+    if (ComboBox_GetCount(hCombo) == 0)
+    {
+        WCHAR sShow[100];
+        WCHAR sHide[100];
+        WCHAR sInactive[100];
+        LoadStringW(NULL, IDS_NOTIF_BEH_SHOW, sShow, _countof(sShow));
+        LoadStringW(NULL, IDS_NOTIF_BEH_HIDE, sHide, _countof(sHide));
+        LoadStringW(NULL, IDS_NOTIF_BEH_HIDE_INACTIVE, sInactive, _countof(sInactive));
```

Comment:
I'm hesitating in using CStringW class for the resource strings here (they would automatically deal correctly with the string sizes). @learn-more , any advice on that?

Reply:
Why would you hestitate?
If you (for some obscure reason) don't want to use CStrings, you can always use the LoadStringW technique of passing it a pointer, and itll point that to the raw resource string (but be careful, it is not always null terminated!)

Reply:
Usage:
```cpp
CStringW str...(MAKEINTRESOURCEW(IDS_...));
```